Dockerfile.worker

```diff
@@ -1,4 +1,4 @@
-FROM redash/redash:latest
+FROM redash/redash:10.1.0.b50633
```

Comment:
This file is still starting a `celery` worker. RQ replaced Celery in v10. Do we need to remove this? 
```
CMD /usr/local/bin/celery worker
```